jobs/utils.py

```diff
@@ -0,0 +1,320 @@
+# Python core Librairies
+import datetime
+import re
+import sqlite3
+
+# Community Librairies
+import pandas as pd
+import requests
+
+from bs4 import BeautifulSoup
+from sqlalchemy import select
+
+
+def check_league(league_name, leagues):
+    '''
+    Check if a league in leagues is in the x, used in a Lambda Function.
+    This is comparing 2 strings between them. Not ideal, but it works.
+    
+    parameters:
+    -----------
+    x: String
+    leagues: list of strings
+    '''
+    for l in leagues:
+        if l in league_name: 
+            return True
+    return False   
+
+def assign_league_id(league_name, leagues_df):
+    '''
+    Assign the correct leagueid used in a Lambda Function.
+    This is comparing 2 strings between them. Not ideal, but it works.
+    
+    parameters:
+    -----------
+    x: String
+    leagues_df: Pandas DataFrame containing leagueid and leaguename
+    '''
+    for l in leagues_df['leaguename'].unique():
+        if l in league_name: 
+            return leagues_df[leagues_df['leaguename'] == l]['id'].values[0]
+    return -1   
```

Comment:
```suggestion
return df.loc[df['leaguename']==league_name,'id'].to_dict().get(1)
```
Return None si le league_name n'est pas trouvé

Reply:
Personellement je préfère retourner "-1". Car je sais que  "-1" c'est une valeur intentionelle, alors que pour None ce n'est pas toujours le cas.